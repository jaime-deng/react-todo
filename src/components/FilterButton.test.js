import renderer from "react-test-renderer"
import FilterButton from "./FilterButton"
import React from "react"
import ReactDOM from "react-dom"

test("App snapshot test", () => {
    const component = renderer.create(<FilterButton />)
    const comp = component.toJSON();
    expect(comp).toMatchSnapshot()
})

it("renders FilterButton component properly", () => {
    const div = document.createElement("div");
    ReactDOM.render(<FilterButton />, div)
})