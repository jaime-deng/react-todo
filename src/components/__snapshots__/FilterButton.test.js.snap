// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<button
  className="btn toggle-btn"
  onClick={[Function]}
  type="button"
>
  <span
    className="visually-hidden"
  >
    Show 
  </span>
  <span />
  <span
    className="visually-hidden"
  >
     tasks
  </span>
</button>
`;
